### ISSUE TITLE
Support in aws aurora serverless data api 

### ISSUE THREAD

**Issue by lielran** [golang-migrate/migrate#294]
aws aurora serverless data api:  https://aws.amazon.com/blogs/aws/new-data-api-for-amazon-aurora-serverless/ enables secure way to access the db from outside of the vpc without the need to configure security groups. this is perfect for ci/cd flows. 

do you think migrate can support that?

**Comment by dhui** [comment]
Yeah, assuming this isn't compatible with how existing DB drivers connect to DBs, feel free to add a new DB driver for this

**Comment by lielran** [comment]
It requires aws-sdk/??? Package. It's all http with json
